src/components/nav.js

```diff
@@ -19,33 +34,95 @@ const Nav = () => {
 
       <div className="Nav--main-links">
         <div className="Nav--item">
-          <Link className="Nav--link" to="/" activeClassName="Nav--link-is-active">Home</Link>
+          <Link
+            className="Nav--link"
+            to="/"
+            activeClassName="Nav--link-is-active"
+          >
+            Home
+          </Link>
         </div>
 
         <div className="Nav--item">
-          <a className="Nav--link" href="https://pages.cloudflare.com">Pages</a>
+          <a className="Nav--link" href="https://pages.cloudflare.com">
+            Pages
+          </a>
         </div>
 
         <div className="Nav--item">
-          <Link className="Nav--link" to="/built-with" partiallyActive={true} activeClassName="Nav--link-is-active">Built with</Link>
+          <Link
+            className="Nav--link"
+            to="/built-with"
+            partiallyActive={true}
+            activeClassName="Nav--link-is-active"
+          >
+            Built with
+          </Link>
         </div>
 
         <div className="Nav--item">
-          <a className="Nav--link" href="https://workers.cloudflare.com/docs">Docs</a>
+          <a className="Nav--link" href="https://workers.cloudflare.com/docs">
+            Docs
+          </a>
         </div>
 
         <div className="Nav--item">
-          <a className="Nav--link" href="https://discord.gg/cloudflaredev">Discord</a>
+          <a className="Nav--link" href="https://discord.gg/cloudflaredev">
+            Discord
+          </a>
         </div>
       </div>
 
+      <div className="Nav--search">
+        {showInput ? (
+          <form
+            id="search-input"
+            className="Nav--search-form"
+            onSubmit={handleOnSubmit}
+          >
+            <div className="Nav--search-icon">
+              <svg
+                viewBox="0 0 16 16"
+                fill="currentcolor"
+                role="img"
+                aria-labelledby="title-6713060681049814"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <title id="title-6713060681049814">
+                  Search icon (depiction of a magnifying glass)
+                </title>
+                <path d="M11.999 10.585l3.458 3.458a1 1 0 01-1.414 1.414L10.585 12a6.5 6.5.0 111.414-1.414zM6.75 11.5a4.75 4.75.0 100-9.5 4.75 4.75.0 000 9.5z"></path>
+              </svg>
+            </div>
+            <input
+              className="Nav--search-input"
+              type="text"
+              value={value}
+              placeholder="Search docs..."
```

Comment:
I think this should say something different. On cloudflare.com (marketing site) it says "Search Cloudflare" -- let's go with that here instead of "docs..."

Reply:
Got will change this to the recommended text.